Tools/LyTestTools/tests/unit/test_launcher_android.py

```diff
@@ -97,17 +97,17 @@ def test_GetPid_PidHasMultipleValues_ReturnPid(self, mock_output):
         assert mock_output.call_count == 2
         assert under_test == '12'
 
-    def test_GenerateMapCommand_HasMapArg_ReturnsMapCommand(self):
-        args_list = ['random_arg', '+map', 'map_name', 'another_arg']
+    def test_GenerateMapCommand_HasLoadLevelArg_ReturnsMapCommand(self):
+        args_list = ['random_arg', '+LoadLevel', 'map_name', 'another_arg']
 
-        under_test = ly_test_tools.launchers.platforms.android.launcher.generate_android_map_command(args_list)
+        under_test = ly_test_tools.launchers.platforms.android.launcher.generate_android_loadlevel_command(args_list)
 
-        assert under_test == 'map map_name'
+        assert under_test == 'LoadLevel map_name'
 
-    def test_GenerateMapCommand_NoMapArg_ReturnsEmptyString(self):
+    def test_GenerateMapCommand_NoLoadLevelArg_ReturnsEmptyString(self):
         args_list = ['random_arg', 'stuff', 'map_name', 'another_arg']
```

Comment:
I wonder if this would make more sense if `'+LoadLevel'` was the last argument in the list.

Reply:
That would cause this test to fail as `generate_android_loadlevel_command` doesn't validate that an argument following "+LoadLevel" exits.   Probably worth adding that guard and test though.

Reply:
Yeah that would be a good test case to add.  As it stands now, the `args_list` here is a random collection of words, so it doesn't really have anything to do with `LoadLevel`.  Maybe change it to `test_GenerateLoadLevelCommand_NoCommand_ReturnsEmptyString` and if you add a new test case for `+LoadLevel` but no arg, that would be `test_GenerateLoadLevelCommand_NoArg_Returns<whatever>`.